test/TokenRequestTest.js

```diff
@@ -185,6 +198,16 @@ contract('TokenRequest', ([rootAccount, ...accounts]) => {
         assert.deepStrictEqual(actualTokens, expectedTokens)
       })
 
+      it('adds ETH to acceptedDepositTokens', async () => {
```

Comment:
```suggestion
      it('adds ETH to accepted deposit token list', async () => {
```
In general we shouldn't make references to contract variables, the descriptions should be in a more general level